settings.yaml

```diff
@@ -0,0 +1,20 @@
+- public_teams:
+  name: public_teams
+  description: "1 if the teams list should be public, 0 if not"
+  value: 0
+- public_judges:
+  name: public_judges
+  description: "1 if the judges list should be public, 0 if not"
```

Comment:
Quick comment, haven't fully reviewed this yet: I think it would be ideal to have a `type` field for cases like this that we set to `boolean`, and then we can render it as a checkbox in the form rather than a number input. I think that would be a bit more of an intuitive UX

If you wanna do it this way as a first iteration and then you (or I) can add the boolean feature on-top of it that's also totally fine

Reply:
I added checkboxes — lmk what you think!